settings.gradle.kts

```diff
@@ -64,6 +64,7 @@ include(":sdk-extensions:incubator")
 include(":sdk-extensions:jaeger-remote-sampler")
 include(":testing-internal")
 include(":animal-sniffer-signature")
+include(":opamp:client")
```

Comment:
If we only anticipate an opamp client artifact, then we might just put this in a directory called `./opamp-client`, instead of `./opamp/client`. Do we anticipate other opamp artifacts?

Reply:
I anticipate a couple of opamp-related artifacts for implementing parts of the API. So for example, opamp data can be sent via HTTP or WebSocket, so for each type of "transport mode" I created an abstraction (similar to HttpSender for exporters) so that people can implement it with their favorite tools, such as okhttp for example. So I was planning to create at least one impl as a separate artifact.